src/vmspawn/vmspawn-scope.c

```diff
@@ -0,0 +1,302 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <stdio.h>
+
+#include "escape.h"
```

Comment:
this is a regular local include, it should be placed among the other regular local includes below. only sd-*.h includes should be separated from the rest.